src/components/Composer/index.js

```diff
@@ -429,14 +429,16 @@ function Composer({
         [style, maxLines, numberOfLinesProp, numberOfLines],
     );
 
+    const isFocused = textInput.current && textInput.current.isFocused();
+
     return (
         <>
             <RNTextInput
                 autoComplete="off"
                 autoCorrect={!Browser.isMobileSafari()}
                 placeholderTextColor={themeColors.placeholderText}
                 ref={(el) => (textInput.current = el)}
-                selection={selection}
+                selection={isFocused ? selection : null}
```

Comment:
@tienifr Could you make the condition only apply to mobile safari? You said that this happened due to the edit composer being re-mount; why do we need to put the logic in the generic composer component?

Reply:
Thanks @tienifr, for the quick update.

It's looking good, but I am trying to understand why we need to put the logic in the generic composer component. Could you explain it?